workspace_host/interface.js

```diff
@@ -461,6 +462,10 @@ async function _allocateContainerPort(workspace) {
   });
 }
 
+/**
+ * @param {object} workspace
+ * @param {function} callback
```

Comment:
I didn't define useful types for these just yet; I only needed this so that the promisified versions would have types that reflect the number of parameters.